### PR TITLE
Setup Windows CI for nightly & distro builds

### LINKED ISSUE #192: Nightly builds are crashing
__Describe the Bug__

The Zeebe Modeler Nightly Builds are not running through currently, cf. https://travis-ci.com/github/zeebe-io/zeebe-modeler/jobs/311138115#L491

__Steps to Reproduce__

Simply go to the [build](https://travis-ci.com/github/zeebe-io/zeebe-modeler/jobs/311138115#L491) and see the nightly builds are crashing.

__Expected Behavior__

It should run through.

__Additional Context__

* [Travis configuration](https://github.com/zeebe-io/zeebe-modeler/blob/develop/.travis.yml)

### PR DESCRIPTION
**Pull request by barmac**
This sets up Windows CI on Travis for both distro and nightly builds. To select proper certificate, we use the cert's fingerprint as the cert subject apparently did not work (cf. https://travis-ci.com/github/zeebe-io/zeebe-modeler/jobs/319668343#L338).

Closes #192

Proof it builds: https://travis-ci.com/github/zeebe-io/zeebe-modeler/jobs/319688610

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
This appears on my Windows machine (x64):

image

**Comment by barmac** [comment]
I updated the PR so that it includes nightly & distro builds on windows.

File: .travis.yml [hunk 1]
```diff
@@ -41,20 +51,30 @@ jobs:
...
-    - stage: "nightly build [windows+linux]"
+    - stage: "nightly build [linux]"
       os: linux
       services: docker
       language: generic
       script:
       - |
         git checkout -b build-nightly
-        ENVS=`env | grep -iE '^(DEBUG|NODE_|ELECTRON_|NPM_|CI|TRAVIS|WIN_CSC_|CSC_|GH_|AWS_|BUILD_)([A-Z]|_)*=' | sed -n '/^[^\t]/s/=.*//p' | sed '/^$/d' | sed 's/^/-e /g' | tr '\n' ' '`
+        ENVS=`env | grep -iE '^(DEBUG|NODE_|ELECTRON_|NPM_|CI|TRAVIS|CSC_|GH_|AWS_|BUILD_)([A-Z]|_)*=' | sed -n '/^[^\t]/s/=.*//p' | sed '/^$/d' | sed 's/^/-e /g' | tr '\n' ' '`
```
Comment (nikku) [inline .travis.yml:61]: We could drop that docker thing all together, if we build each distribution on its native platform.

File: .travis.yml [hunk 2]
```diff
@@ -12,19 +12,29 @@ jobs:
...
       - npx codecov
-    - stage: "distro [windows+linux]"
+    - stage: "distro [linux]"
       os: linux
       services: docker
       language: generic
       script:
       - |
-        ENVS=`env | grep -iE '^(DEBUG|NODE_|ELECTRON_|NPM_|CI|TRAVIS|WIN_CSC_|CSC_|GH_|AWS_|BUILD_)([A-Z]|_)*=' | sed -n '/^[^\t]/s/=.*//p' | sed '/^$/d' | sed 's/^/-e /g' | tr '\n' ' '`
+        ENVS=`env | grep -iE '^(DEBUG|NODE_|ELECTRON_|NPM_|CI|TRAVIS|CSC_|GH_|AWS_|BUILD_)([A-Z]|_)*=' | sed -n '/^[^\t]/s/=.*//p' | sed '/^$/d' | sed 's/^/-e /g' | tr '\n' ' '`
```
Comment (nikku) [inline .travis.yml:21]: Same here: Drop that `docker` thing all together.

**Comment by nikku** [comment]
To be discussed: Port the change to Camunda Modeler, too? Personally I'd do it.

**Comment by barmac** [comment]
Umm @nikku was it my missclick or did you merge this? :D

File: .travis.yml [hunk 3]
```diff
@@ -12,19 +12,22 @@ jobs:
...
+      script:
       - |
-        ENVS=`env | grep -iE '^(DEBUG|NODE_|ELECTRON_|NPM_|CI|TRAVIS|WIN_CSC_|CSC_|GH_|AWS_|BUILD_)([A-Z]|_)*=' | sed -n '/^[^\t]/s/=.*//p' | sed '/^$/d' | sed 's/^/-e /g' | tr '\n' ' '`
-        docker run $ENVS --rm \
-          -v ${PWD}:/project \
-          -v ~/.cache/electron:/root/.cache/electron \
-          -v ~/.cache/electron-builder:/root/.cache/electron-builder \
-          electronuserland/builder@sha256:b50e44a98ffe9f716b2f66c51b7898404670c8b74c8c3fc2b152c53f8af60513 \
-          /bin/bash -c "npm i && npm run build -- --win --linux --publish"
+        git checkout -b build-nightly
```
Comment (nikku) [inline .travis.yml:27]: Windows distro operates on `build-nightly` branch? Is that a typo?

Comment (barmac) [reply .travis.yml:27]: This is a typo.

**Comment by barmac** [comment]
Anyway, I thought I merged this by mistake so I reverted the change before noticing your activity. I will create a separate PR with same content + typo fix.

**Comment by pinussilvestrus** [comment]
I approved it before Nico gave his feedback, sorry!

**Comment by barmac** [comment]
All right!

**Comment by nikku** [comment]
Looking into the future we could consider moving all distro jobs to the `distro` stage, too: https://docs.travis-ci.com/user/build-stages#naming-your-jobs-within-build-stages

This would speed up the build significantly.

Right now we build each distribution one after the other.

If all jobs run within the same stage, they run in paralle.

**Comment by nikku** [comment]
Too fast you guys are. Get some sleep and relaxation.

It's just poor me still working as I've spent the afternoon in the :sun_with_face: .

**Comment by barmac** [comment]
Let's finish work here: https://github.com/zeebe-io/zeebe-modeler/pull/209

**Comment by nikku** [comment]
PR ALL THINGS!

**Comment by pinussilvestrus** [comment]
🍾🌴

**Pull request merged**